sunpy/coordinates/utils.py

```diff
@@ -265,3 +265,74 @@ def coordinates(self, points=None):
                         obstime=self.obstime,
                         observer=self.observer,
                         frame=Heliocentric).transform_to(self.start_frame)
+
+
+def get_rectangle_coordinates(bottom_left, *, top_right = None, width: u.deg = None, height: u.deg = None):
```

Comment:
What's the thinking behind the bare asterisk here?  Why not allow `top_right` to be a positional argument?

Reply:
My thinking was that allowing a second positional argument is ambiguous between the two inputs. See my long comment here: https://github.com/sunpy/sunpy/pull/3677#issuecomment-575908202 for more thoughts on why this function signature.

Reply:
It's not that ambiguous here because there are different numbers of arguments, but sure.

The docstring should show example call strings, which I think would help greatly in preparing users..